db/seeds.rb

```diff
@@ -18,6 +18,13 @@ def format_errors(model = nil)
     ForemanTasks.sync_task(::Actions::Katello::Organization::Create, org) unless org.library
     User.current = nil
   end
+  if Setting['db_pending_seed']
```

Comment:
Ah wasn't aware of that.  If you have any other thoughts I'm open to ideas but I think I'm okay with that as well.  The value to most initial users is high enough i think it is worth it.